portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsRulesActions.testcase

```diff
@@ -431,6 +520,79 @@ definition {
 			successPageTitle = "Thank you.");
 	}
 
+	@description = "Verify that a Rule with 'Require' action can be created"
+	@priority = "4"
+	test CanCreateARequireRule {
+		property portal.acceptance = "true";
+		property portal.upstream = "true";
+
+		var portalURL = PropsUtil.get("portal.url");
```

Comment:
@leticiachiu, where this variable is being used?

Reply:
Oh sorry, I just moved the old test to the file, but I'll delete this line now :)